src/org/etools/j1939_84/bus/j1939/packets/DM31DtcToLampAssociation.java

```diff
@@ -28,6 +28,14 @@ public DM31DtcToLampAssociation(Packet packet) {
         super(packet, new J1939DaRepository().findPgnDefinition(PGN));
     }
 
+    public static DM31DtcToLampAssociation create(int sourceAddress, List<DTCLampStatus> lampStatuses) {
```

Comment:
You can use var args here

`create(int address, DTCLampStatus... lampStatuses)`